vespa-athenz/src/main/java/com/yahoo/vespa/athenz/identity/SiaIdentityProvider.java

```diff
@@ -29,24 +33,31 @@
 
     private static final Duration REFRESH_INTERVAL = Duration.ofHours(1);
```

Comment:
It's still 1 hour refresh interval?

Reply:
Yes, the credentials are read from disk every hour, although `sia` daemon is typically configured to update once every day.